.circleci/config.yml

```diff
@@ -49,7 +49,9 @@ commands:
         steps:
             - run:
                 name: Build and run unit tests
-                command: npm run build
```

Comment:
Thanks, this always hurt my eyes. Agree to separate to build and test step.

Reply:
agree